### ISSUE TITLE
Yarn complains about package-lock.json existing

### ISSUE THREAD

**Issue by JPTIZ** [codeisscience/journal-policy-tracker-frontend#5]
I've run `yarn install` in order to install dependencies, and in the first lines of yarn logs:

```log
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this
 warning, remove package-lock.json.  
```

Focusing on "Your project contains lock files generated by tools other than Yarn", maybe it is because of a previous manual npm run?

**Comment by yochannah** [comment]
note - try npm install. if this works, we should update the docs to say start with npm install

**Comment by yochannah** [comment]
quick check in - @ikayz is this all fixed now?

**Comment by JPTIZ** [comment]
Since there's still both `package-lock.json` and `yarn.lock`, yarn would still complain about the other lock file. My suggestion is to stick with one package manager only in a project (either npm or yarn), unless we keep a strategy to confirm we support both at each release.

**Comment by ikayz** [comment]
@yochannah yes. I removed the existing package-lock and ran npm install instead to fix the issue. It's fixed in the commit I'll push today. @JPTIZ true, I've gone that route

**Comment by JPTIZ** [comment]
Closed by #17 :)
